packages/api-server/src/index.ts

```diff
@@ -3,12 +3,13 @@
 import { hideBin } from 'yargs/helpers'
 import yargs from 'yargs/yargs'
 
+// @ts-expect-error TODO
+import * as webServerCLIConfig from '@redwoodjs/web-server/cli-config'
```

Comment:
I tried copying the type setup in `@redwoodjs/vite` which also has these entry points but I can't get it work; TS thinks this is untyped still